docs/guide/docs/channels/converse.md

```diff
@@ -3,76 +3,189 @@ id: converse
 
-## Usage (Public API)
+## Public API
 
-`POST /api/v1/bots/{botId}/converse/{userId}` where **userId** is a unique string identifying a user that chats with your bot (**botId**), you can use random string for **userId**, also don't forget to set the `Content-Type` HTTP header to `application/json`.
+The Public API allows you to interact with your chatbot and get an answer synchronously. In other words, you can hold a 'conversation' with your chatbot using the converse API. The only difference is that responses are delivered to the calling application as JSON and not through a channel with a GUI.
 
-### Request Body
+### Request Header
+
+To 'speak' with your chatbot via the converse API, make a call with the HTTP header `Content-Type` set to `application/json`)
 
+### Request URL
+Make a post request to the converse API. below is an example of the request format
+```json
+POST <your-bot-url.com>/api/v1/bots/{botId}/converse/{userId}
+```
+ -Replace **userId** with any unique string to represent a user chatting with your chatbot (**botId**).
+ -For **botId**, go to `\data\bots\BOTNAME`, then open the `bot.config.json` file where the last two lines will specify bot id.
+
+### Request Body
+In the request body, place the message you are sending to your chatbot as a JSON object.
 ```json
 {
   "type": "text",
-  "text": "Hey!"
+  "text": "Google Stock Price"
+}
+```
+
+### Request Response
+The response you get from this API call is the bot's response to this message in the website embedded (or any other) channel. Below is a typical response.
+```json
+{
+    "responses": [
+        {
+            "type": "typing",
+            "value": true
+        },
+        {
+            "type": "text",
+            "markdown": true,
+            "text": "NASDAQ: GOOGL**2,385.50 USD** Get stock recommendations [here](http:somesite.com) "
+        }
+    ]
 }
 ```
 
-## Usage (Debug API)
+## Debug API
+There's also a secured route (requires authentication to Botpress to consume this API). Using this route, you can request more data to be included in your response using the `include` query params separated by commas.
 
-There's also a secured route (requires authentication to Botpress to consume this API). Using this route, you can include more data to your response by using the `include` query params separated by commas.
+### JWT Token
+To access this route, you need a jwt token. This requirement is in place because sensitive information destined for the chatbot back-end user is accessible via this route. For example, you can access the chatbot's decision logic.  
 
-### Example
+#### Request
+Here is a sample request using cURL to get this token:
 
+```js
+curl --location --request POST 'http://<your.botpress.server.com>/api/v1/auth/login/basic/default' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "email": "<YOUR-EMAIL>",
+    "password": "<YOUR-PASSWORD>"
+}'
```

Comment:
I'm not sure this works anymore since we added csrf tokens for enhanced security on the admin & studio applications. Do you confirm that this works on the latest version with the default botpress config ?